Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -153,8 +177,14 @@ div#attachment-save-area {
 
 button#attachment-save-button {
 #if (defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
+#if (defined(WTF_PLATFORM_VISION) && WTF_PLATFORM_VISION)
+    width: 28px;
+    height: 28px;
+    padding: 8px;
```

Comment:
Please use `mask-size` on `button#attachment-save-button::before` to set the icon size like other platforms